services/openshift-service-impl/src/main/resources/pipeline-template.yml

```diff
@@ -16,7 +21,12 @@ objects:
   kind: BuildConfig
   metadata:
     name: ${PROJECT}
+    annotations:
+      pipeline.alpha.openshift.io/uses: '[{"name": "jenkins", "namespace": "", "kind":
+        "DeploymentConfig"}]'
   spec:
+    triggers:
+    - type: ConfigChange
```

Comment:
@edewit And I guess this is the necessary configuration for making things start automatically?